### PR TITLE
Brotli compression

### PR DESCRIPTION
**Pull request by stefholbrook**
This PR adds Brotli for css and js compression. 

This implementation uses  the [Brotli](https://hex.pm/packages/brotli) package. This seemed to fit our needs, had a straightforward implementation, and was the most widely used. 

Issue: [159](https://github.com/BeaconCMS/beacon/issues/159)

### CODE REVIEW

File: lib/beacon/runtime_css.ex [hunk 1]
```diff
@@ -21,10 +21,17 @@ defmodule Beacon.RuntimeCSS do
...
-    true = :ets.insert(:beacon_assets, {{site, :css}, {hash, css, compressed}})
-    :ok
+
+    case :brotli.encode(css) do
+      {:ok, compressed} ->
+        hash = Base.encode16(:crypto.hash(:md5, css), case: :lower)
+        true = :ets.insert(:beacon_assets, {{site, :css}, {hash, css, compressed}})
+        :ok
+
+      error ->
```
Comment (stefholbrook) [inline lib/beacon/runtime_css.ex:31]: Not sure if we care about the error here, but included it just in case. Happy to remove it.

File: .github/workflows/elixir.yml [hunk 2]
```diff
@@ -21,9 +21,11 @@ jobs:
         include:
           - elixir: "1.13.0"
             otp: "23"
+            rebar3_version: '3.15.2'
```
Comment (leandrocp) [inline .github/workflows/elixir.yml:24]: Do we still need these changes?

File: lib/beacon/runtime_css.ex (hunk 1 above)
Comment (leandrocp) [reply lib/beacon/runtime_css.ex:31]: That's fine. Raising here means that a site wouldn't load but on the other hand starting a site without styles and JS isn't a good option either.
The alternative is fall backing to gzip or uncompressed but it's not worth it unless we start having issues.

File: priv/static/beacon_admin.css
Comment (leandrocp) [inline priv/static/beacon_admin.css:1]: Can you please revert the changes on these static files?

**Review by leandrocp — CHANGES_REQUESTED** [review]
Nice one thanks!

**Review by xn — APPROVED** [review]

**Review by leandrocp — APPROVED** [review]
Thanks! 🎉
